internal/services/iothub/iothub_endpoint_cosmosdb_account_resource.go

```diff
@@ -131,6 +132,14 @@ func (r IotHubEndpointCosmosDBAccountResource) Arguments() map[string]*pluginsdk
 			ConflictsWith: []string{"identity_id"},
 			RequiredWith:  []string{"primary_key"},
 		},
+
+		// NOTE: O+C : required since this property would always be set even if it isn't specified in the tf config, otherwise it would cause a diff and break existing users
+		"subscription_id": {
+			Type:         pluginsdk.TypeString,
+			Optional:     true,
+			Computed:     true,
+			ValidateFunc: validation.IsUUID,
```

Comment:
```suggestion
		"subscription_id": {
			Type:         pluginsdk.TypeString,
			Optional:     true,
			// NOTE: O+C : required since this property would always be set even if it isn't specified in the tf config, otherwise it would cause a diff and break existing users
			Computed:     true,
			ValidateFunc: validation.IsUUID,
```
Could the note be moved between `Optional` and `Computed`? We're trying to make this the standard.

Reply:
Updated

Reply:
Updated